src/Kodify/BlogBundle/Tests/Controller/CommentsControllerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Kodify\BlogBundle\Tests\Controller;
+
+use Kodify\BlogBundle\Entity\Comment;
+use Kodify\BlogBundle\Entity\Post;
+use Kodify\BlogBundle\Entity\Author;
+use Kodify\BlogBundle\Tests\BaseFunctionalTest;
+
+class CommentsControllerTest extends BaseFunctionalTest
+{
+
+    protected function createComments($count)
+    {
+        $author = new Author();
+        $author->setName('Author');
+        $this->entityManager()->persist($author);
+        $this->entityManager()->flush();
+
+        $post = new Post();
+        $post->setTitle('Title');
+        $post->setContent('Content');
+        $post->setAuthor($author);
+        $this->entityManager()->persist($post);
+        $this->entityManager()->flush();
+
+        for ($i = 0; $i < $count; ++$i)
+        {
+            $comment = new Comment();
+            $comment->setContent('Comment content ' . $i);
+            $comment->setAuthor($author);
+            $comment->setPost($post);
+            $comment->setCreatedAt( new \DateTime() );
+            $comment->setUpdatedAt( new \DateTime() );
```

Comment:
You should try to be consistent with the code style, in example in this line you let 1 space after and before parentheses, and in other places there's no spacing. Also, you have to be aware when you open an { of an "if" or "for", the one from this class is opened in a new line and in other places is in the same.

FYI, here in Kodify we do use the Symfony Coding Standard http://symfony.com/doc/current/contributing/code/standards.html